Update sync-process-explained - queuing syncs.md (#619)
codatio/codat-docs: docs/sync-for-expenses/sync-process/sync-process-explained.md

```
A `syncId` will be returned to the response payload.

> Multiple syncs can be initiated at once. The queueing of these syncs will be managed on the Codat side and pushed to the accounting platform.

Reviewer: ```suggestion
You can initiate multiple syncs at once. Codat will manage the queueing of these syncs and push the data to the accounting platform.
```
